Makefile

```diff
@@ -127,7 +127,7 @@ lint:
 
 .PHONY: audit
 audit:
-	$(CARGO) audit --ignore RUSTSEC-2020-0123
+	$(CARGO) audit --ignore RUSTSEC-2020-0123 --ignore RUSTSEC-2020-0146
```

Comment:
If you merge with `dev` you shouldn't have to do this.

Reply:
out of scope for this PR